src/angular-selector.css

```diff
@@ -5,8 +5,8 @@
 
 /* Selector */
 [selector] { display: none; position: relative; }
-[selector].selector { display: block; }
-.selector { font-size: 1em; line-height: normal; color: #495c68; text-shadow: 0px 1px 0px rgba(255, 255, 255, .5); }
+[selector].selector-item { display: block; }
```

Comment:
Change all `selector-item` class to `selector-container`, we will add this class only to the main container.